db/migrate/20150712193314_index_cleanup.rb

```diff
@@ -0,0 +1,15 @@
+class IndexCleanup < ActiveRecord::Migration
+  def change
+    # Extremely common operations
+    add_index :reviews, :user_id
+    add_index :reviews, :anime_id
+    add_index :recommendations, :user_id
+
+    # WTF?
+    remove_index :users, :waifu_or_husbando
```

Comment:
Since there's a work on waifu/husbando on #375 (it's still active?) I think it's okay to remove this without the *wtf?* comment… :P